pkg/apply/apply.go

```diff
@@ -51,15 +56,26 @@ func ApplyObject(ctx context.Context, client *cnoclient.Client, obj Object, subc
 	// It isn't allowed to send ManagedFields in a Patch.
 	obj.SetManagedFields(nil)
 
+	anno := obj.GetAnnotations()
+	if anno != nil {
+		if _, exists := anno[names.CopyFromAnnotation]; exists {
+			var err error
+			obj, err = getObjectCopy(ctx, obj, client)
+			if err != nil {
+				return err
```

Comment:
Wrap this error with some useful text: `return fmt.Errorf("failed to retrieve copy-from object: %w", err)`